lib/oli_web/components/delivery/instructor_dashboard.ex

```diff
@@ -169,7 +169,7 @@ defmodule OliWeb.Components.Delivery.InstructorDashboard do
 
   def header(assigns) do
     ~H"""
-    <div class="w-full bg-delivery-instructor-dashboard-header text-white border-b border-slate-600">
+    <div class="w-full bg-delivery-instructor-dashboard-header text-white border-b border-slate-600 fixed top-0 z-50">
```

Comment:
If we take the `fixed` class approach we should add a margin-top to `instructor_dashboard.html.heex` to avoid having the course title being overlapped by the header.

https://github.com/Simon-Initiative/oli-torus/assets/74839302/07935833-e161-482e-ba08-c6ec87a30f20



With this approach, we get an animation when scrolling all the way up to the top, where a white background is seen by a fraction of a sec. (maybe [overscroll-behaviour ](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior) can fix it, not sure TBH).


Another approach may be using the `sticky` class, ensuring adapting the top position of the flash message to avoid being overlapped by the header (you've already adjusted that top position).

Reply:
I have made another iteration. This time, the only minor issue is that when reloading the page, the Cookie Preferences briefly appear at the top for a fraction of a second before disappearing. What do you think? Is that or the white space as you mentioned earlier in between the menu bar and the content.

Reply:
No, the white space I was refering to is the one at second 23 in the attached video. Here is a screenshot

![image](https://github.com/Simon-Initiative/oli-torus/assets/74839302/9c9438b8-939c-4438-a020-9cf2ec851565)

I pulled the changes and saw the issue with the cookie preferences. It looks really weird. I think we should fix that for sure